docs/source/usage.rst

```diff
@@ -125,5 +125,50 @@ To keep things clean and simple, we advise a few tips:
   for more information.
 
 
+5. Tagging versions of your code
+--------------------------------
+
+Tagging versions of your code is a good way to keep references
+to specific versions of your code that don't change.
+This is especially useful when you want to make a new code
+release. The ``cookiecutter`` uses `versioningit`_ to
+automatically determine your package's version based on
+git tags.
+
+By default, versioning will start from ``0.0.0``.
+You can install versioningit_ to check versioning it from commandline:
```

Comment:
```suggestion
You can install versioningit_ to check the current versioningit output from commandline:
```
Or something like that, wasn't too such what was meant by "check versioning it"

Reply:
Thanks for fixing that, I don't think that meant anything, my brain just stopped working or something